spark/src/main/java/org/apache/zeppelin/spark/DepInterpreter.java

```diff
@@ -183,7 +183,9 @@ public InterpreterResult interpret(String st, InterpreterContext context) {
 
     if (sparkInterpreter != null && sparkInterpreter.isSparkContextInitialized()) {
       return new InterpreterResult(Code.ERROR,
-          "Must be used before SparkInterpreter (%spark) initialized");
+          "Must be used before SparkInterpreter (%spark) initialized\n" +
+          "Hint: put this cell before any spark/scala code or/and " +
+          "restart Zeppelin" );
```

Comment:
@xhudik Zeppelin has Notebooks and Paragraphs. Paragraph is the term we use for cells or sections we have on the UI. Jupyter and Beaker Notebook uses the term Notebooks and Cells. Please take a look at this screenshot.

<img width="550" alt="screen shot 2015-11-05 at 4 21 02 pm" src="https://cloud.githubusercontent.com/assets/2031306/10966384/40897fae-83d9-11e5-90f4-df3f768b20bb.png">